src/display/api.js

```diff
@@ -2800,7 +2804,9 @@ class WorkerTransport {
         }
       }
       this.commonObjs.clear();
-      this.fontLoader.clear();
+      if (cleanupFontLoader) {
```

Comment:
Following the above:
```suggestion
      if (!keepFonts) {
```